docs/build-instructions/windows.md

```diff
@@ -48,6 +48,17 @@ This will create the Atom application in the `out\Atom` folder as well as copy i
   * `--no-install` - Skips the installation task after building.
   * `--verbose` - Verbose mode. A lot more information output.
 
+### Running tests
+
+In order to run tests from command line you need `apm`, available after you install Atom or after you build that from scratch. If you installed it, run the following commands (assuming `C:\atom` is the root path of your Atom sources):
```

Comment:
* build that from scratch &rarr; build from source
* root path of your Atom sources &rarr; root of your Atom repository